src/Cli/dotnet/Commands/Tool/Uninstall/ToolUninstallCommandParser.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.DotNet.Cli.Commands.Tool.Uninstall;
 
 internal static class ToolUninstallCommandParser
 {
-    public static readonly CliArgument<string> PackageIdArgument = ToolInstallCommandParser.PackageIdArgument;
+    public static readonly CliArgument<(string PackageId, string Version)> PackageIdArgument = CommonArguments.PackageIdentityArgument();
```

Comment:
Does it make sense to specify a version when uninstalling a tool?  It looks like we don't currently support a version option here, so I don't think we need to add the `package@version` syntax in this case.